### PR TITLE
Check the content of the file containing annotations

### PR DESCRIPTION
**Pull request by amelieraymond**
creation of a function "check_annotation" to check if the content of a given file matches some regex.

### CODE REVIEW

File: python/test.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import re
```
Comment (rjoganah) [inline python/test.py]: For testing purpose, you should use a unittest framework, such as "https://docs.python.org/fr/3/library/unittest.html"

File: python/dataStructure_process.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+'''
+    TRAIN TEST SPLIT
+    Split the dataset in two according to a specified proportion
+    output : 
+        train, test : two datasets
+'''
+def train_test_split(data, training_proportion):
```
Comment (rjoganah) [inline python/dataStructure_process.py]: Same comment as in the last pull request, you might use https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html or add random seed and shuffle capability.

File: python/dataStructure_process.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+'''
```
Comment (rjoganah) [inline python/dataStructure_process.py]: You should not mix camelcase and snake case in file naming

File: python/data_process.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
+import re
+from os import chdir, getcwd
+wd=getcwd()
+chdir(wd)
```
Comment (rjoganah) [inline python/data_process.py]: It's a better practice to use os.path.join(directory, filename) in the code, instead of changing the wd in the script directly which might be more prone to errors.

File: python/test.py (hunk 1 above)
Comment (amelieraymond) [reply python/test.py]: Good idea thanks, I didn't know this framework

File: python/dataStructure_process.py (hunk 2 above)
Comment (amelieraymond) [reply python/dataStructure_process.py]: Yes, thanks!
For now, we use this really basic version as we are still trying to find a way (like using hash of the train/test data?) to be certain that several models have been trained and tested on the same data sets, in case someone imports a trained model and want to compare its performance to a new trained model.

File: python/dataStructure_process.py (hunk 3 above)
Comment (amelieraymond) [reply python/dataStructure_process.py]: Thank you, I will change that!

File: python/data_process.py (hunk 4 above)
Comment (amelieraymond) [reply python/data_process.py]: Thank you, I will change that!
